cdc/sorter/leveldb/system/system.go

```diff
@@ -68,10 +70,14 @@ func NewSystem(dir string, memPercentage float64, cfg *config.DBConfig) *System
 		WorkerNumber(cfg.Count).Build()
 	compactSystem, compactRouter := actor.NewSystemBuilder("sorter-compactor").
 		WorkerNumber(cfg.Count).Build()
+	writerSystem, writerRouter := actor.NewSystemBuilder("sorter-writer").
```

Comment:
we have  `sorter-db` and `sorter-writer`,   can you add some comments about `sorter-db`